models/data/IPortfolio.ts

```diff
@@ -1,4 +1,36 @@
 /** Portfolio data */
+export interface IWorkHistory {
+  position?: string;
+  company?: string;
+  dateWorked?: string;
+  description?: string;
+}
+export interface IProjects {
+  title?: string;
+  picture?: string;
+  summary?: string;
+  viewCode?: string;
+  liveDemo?: string;
+}
+
+export interface ISkills {
+  tech?: string[];
+  soft?: string[];
+}
+
+export interface IContact {
+  codeRepoLink?: string;
+  linkedin?: string;
+  email?: string;
+}
+
+export interface IMessage {
```

Comment:
Could you remove this interface for now, and we can have a separate task for messaging? Portfolio interface is for portfolio content, Message interface should be totally separate